### PR TITLE
Home task webpack

### PR DESCRIPTION
**Pull request by LxndrKrchvtsv**
This PL contains 1 and 2 tasks.

### CODE REVIEW

**Review by Dmitry-White — CHANGES_REQUESTED** [review]
Nice work!
Left a couple of comments for further improvements

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "trailingComma": "es5",
+  "printWidth": 140,
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 4,
```
Comment (Dmitry-White) [inline .prettierrc:6]: Sure you want to have so much indentation going forward? :)
React JSX tends to get pretty nested :)

**Review by Dmitry-White — APPROVED** [review]
Great job!
Approving upfront, though left some comments/suggestions

File: .prettierrc (hunk 1 above)
Comment (LxndrKrchvtsv) [reply .prettierrc:6]: @Dmitry-White I think, yes, because it's just more comfortable to me.) I don't like tags with width 2 spaces.)

**Pull request merged**